lib/screens/explore_screens/flag/components/remove_flag_info_dialog.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:seeds/components/custom_dialog.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/i18n/profile_screens/profile/profile.i18n.dart';
+import 'package:seeds/screens/explore_screens/flag/interactor/viewmodels/flag_bloc.dart';
+
+class RemoveFlagInfoDialog extends StatelessWidget {
+  final String userAccount;
+
+  const RemoveFlagInfoDialog({required this.userAccount, Key? key}) : super(key: key);
```

Comment:
You can avoid the required keyword, since its just one paramer `(this.userAccount, {Key? key})`